tests/test_cli.py

```diff
@@ -0,0 +1,343 @@
+import os
+import pytest
+import importlib_resources
+from click.testing import CliRunner
+
+from genet import cli
+
+
+GENET_ROOT_DIR = importlib_resources.files("genet").parent
+EXAMPLE_DATA_DIR = GENET_ROOT_DIR / 'example_data'
+
+EXAMPLE_NETWORK = EXAMPLE_DATA_DIR / 'pt2matsim_network' / 'network.xml'
+EXAMPLE_SCHEDULE = EXAMPLE_DATA_DIR / 'pt2matsim_network' / 'schedule.xml'
+EXAMPLE_VEHICLES = EXAMPLE_DATA_DIR / 'pt2matsim_network' / 'vehicles.xml'
+PROJECTION = 'epsg:27700'
+
+@pytest.fixture(scope="function")
+def invoke_runner_and_check_files(tmpdir_factory):
+    outdir = tmpdir_factory.mktemp("outdir")
+
+    def _invoke_runner_and_check_files(
+        func: str, args: list[str], expected_files: list = [], not_expected_files: list = []
+    ):
+        runner = CliRunner()
```

Comment:
in the original scripts tests there was a check before running the process that the expected files do not already exist, can you add it here?